### PR TITLE
[RS][DDCE-1391] Play 2.6 upgrade.

### PR DESCRIPTION
**Pull request by RafSobol**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# DDCE-StoryNumberHere
```
Comment (abhijeetardale) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: do we really need, this, i never seen people updating it?

File: app/uk/gov/hmrc/payedesstub/controllers/DocumentationController.scala [hunk 2]
```diff
@@ -16,24 +16,25 @@
...
 import uk.gov.hmrc.payedesstub.config.ApplicationConfig
 import uk.gov.hmrc.payedesstub.views.txt
-import uk.gov.hmrc.play.bootstrap.controller.BaseController
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
 @Singleton
-class DocumentationController @Inject()(httpErrorHandler: HttpErrorHandler, appConfig: ApplicationConfig)
-  extends AssetsBuilder(httpErrorHandler) with BaseController {
+class DocumentationController @Inject()(appConfig: ApplicationConfig,
+                                        assets: Assets,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/controllers/DocumentationController.scala:28]: 2 space formating please

File: app/uk/gov/hmrc/payedesstub/controllers/IndividualBenefitsController.scala [hunk 3]
```diff
@@ -23,14 +23,17 @@ import play.api.mvc._
...
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.ExecutionContext
 
 @Singleton
 class IndividualBenefitsController @Inject()(val scenarioLoader: ScenarioLoader,
-                                             val service: IndividualBenefitsSummaryService)
-  extends BaseController with HeaderValidator {
+                                             val service: IndividualBenefitsSummaryService,
+                                             val cc: ControllerComponents)
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/controllers/IndividualBenefitsController.scala:33]: 2 space alignment please

File: app/uk/gov/hmrc/payedesstub/controllers/IndividualEmploymentController.scala [hunk 4]
```diff
@@ -22,14 +22,17 @@ import play.api.mvc._
...
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.ExecutionContext
 
 @Singleton
 class IndividualEmploymentController @Inject()(val scenarioLoader: ScenarioLoader,
-                                               val service: IndividualEmploymentSummaryService)
-  extends BaseController with HeaderValidator {
+                                               val service: IndividualEmploymentSummaryService,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/controllers/IndividualEmploymentController.scala:31]: 2 space alignment please

File: app/uk/gov/hmrc/payedesstub/controllers/IndividualIncomeController.scala [hunk 5]
```diff
@@ -18,18 +18,21 @@ package uk.gov.hmrc.payedesstub.controllers
...
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.ExecutionContext
 
 @Singleton
 class IndividualIncomeController @Inject()(val scenarioLoader: ScenarioLoader,
-                                           val service: IndividualIncomeSummaryService)
-  extends BaseController with HeaderValidator {
+                                           val service: IndividualIncomeSummaryService,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/controllers/IndividualIncomeController.scala:31]: 2 space alignment please

File: app/uk/gov/hmrc/payedesstub/controllers/IndividualTaxController.scala [hunk 6]
```diff
@@ -18,18 +18,21 @@ package uk.gov.hmrc.payedesstub.controllers
...
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.ExecutionContext
 
 @Singleton
 class IndividualTaxController @Inject()(val scenarioLoader: ScenarioLoader,
-                                        val service: IndividualTaxSummaryService)
-  extends BaseController with HeaderValidator {
+                                        val service: IndividualTaxSummaryService,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/controllers/IndividualTaxController.scala:31]: 2 space alignment please

File: app/uk/gov/hmrc/payedesstub/services/IndividualBenefitsSummaryService.scala [hunk 7]
```diff
@@ -26,7 +25,10 @@ import scala.concurrent.Future
 @Singleton
 class IndividualBenefitsSummaryService @Inject()(val repository: IndividualBenefitsRepository) {
 
-  def create(utr: String, taxYear: String, individualBenefitsResponse: IndividualBenefitsResponse): Future[IndividualBenefits] = {
+  def create(utr: String,
+             taxYear: String,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/services/IndividualBenefitsSummaryService.scala:29]: 2 space alignment please

File: app/uk/gov/hmrc/payedesstub/services/IndividualEmploymentSummaryService.scala [hunk 8]
```diff
@@ -26,7 +25,10 @@ import scala.concurrent.Future
 @Singleton
 class IndividualEmploymentSummaryService @Inject()(val repository: IndividualEmploymentRepository) {
 
-  def create(utr: String, taxYear: String, individualEmploymentResponse: IndividualEmploymentResponse): Future[IndividualEmployment] = {
+  def create(utr: String,
+             taxYear: String,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/services/IndividualEmploymentSummaryService.scala:29]: 2 space alignment please

File: app/uk/gov/hmrc/payedesstub/services/IndividualIncomeSummaryService.scala [hunk 9]
```diff
@@ -25,7 +25,10 @@ import scala.concurrent.Future
 @Singleton
 class IndividualIncomeSummaryService @Inject()(val repository: IndividualIncomeRepository) {
 
-  def create(utr: String, taxYear: String, individualIncomeResponse: IndividualIncomeResponse): Future[IndividualIncome] = {
+  def create(utr: String,
+             taxYear: String,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/services/IndividualIncomeSummaryService.scala:29]: 2 space alignment please

File: app/uk/gov/hmrc/payedesstub/services/IndividualTaxSummaryService.scala [hunk 10]
```diff
@@ -26,7 +25,10 @@ import scala.concurrent.Future
 @Singleton
 class IndividualTaxSummaryService @Inject()(val repository: IndividualTaxRepository) {
 
-  def create(utr: String, taxYear: String, individualTaxResponse: IndividualTaxResponse): Future[IndividualTax] = {
+  def create(utr: String,
+             taxYear: String,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/services/IndividualTaxSummaryService.scala:29]: 2 space alignment please

File: app/uk/gov/hmrc/payedesstub/services/TaxHistoryService.scala [hunk 11]
```diff
@@ -26,7 +26,10 @@ import scala.concurrent.Future
 @Singleton
 class TaxHistoryService @Inject()(val repository: TaxHistoryRepository) {
 
-  def create(nino: Nino, taxYear: TaxYear, taxHistoryResponse: String): Future[TaxHistory] = {
+  def create(nino: Nino,
+             taxYear: TaxYear,
```
Comment (abhijeetardale) [inline app/uk/gov/hmrc/payedesstub/services/TaxHistoryService.scala:30]: 2 space alignment please

File: build.sbt [hunk 12]
```diff
@@ -1,75 +1,70 @@
...
+def itTestFilter(name: String): Boolean = name startsWith "it"
+
 lazy val compile = Seq(
-  "uk.gov.hmrc" %% "simple-reactivemongo" % "7.30.0-play-25",
-  "uk.gov.hmrc" %% "bootstrap-play-25" % "5.4.0",
-  "uk.gov.hmrc" %% "domain" % "5.10.0-play-25"
+  "uk.gov.hmrc" %% "simple-reactivemongo" % "7.31.0-play-26",
+  "uk.gov.hmrc" %% "bootstrap-backend-play-26" % "3.2.0",
+  "uk.gov.hmrc" %% "domain" % "5.10.0-play-26",
+  compilerPlugin("com.github.ghik" % "silencer-plugin"     % silencerVersion cross CrossVersion.full),
```
Comment (abhijeetardale) [inline build.sbt:18]: remove additional spaces please

**Review by abhijeetardale — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (RafSobol) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: I'd say it's worth keeping just for new joiners to use

**Review by abhijeetardale — APPROVED** [review]

**Pull request merged**